includes/server-timing/load.php

```diff
@@ -48,6 +50,8 @@ function perflab_server_timing() {
 
 	return $server_timing;
 }
+
+// @phpstan-ignore-next-line
```

Comment:
Because PHPStan identifies correctly that this function is used as an action, and not a filter. So it shouldn't return anything.